smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/category/CategoryController.java

```diff
@@ -1,5 +1,74 @@
-package com.scm.smartcontactmanager.controller.category;
+ package com.scm.smartcontactmanager.controller.category;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.scm.smartcontactmanager.controller.category.request.CategoryRequestDto;
+import com.scm.smartcontactmanager.controller.category.request.ContactIds;
+import com.scm.smartcontactmanager.entity.category.Category;
+import com.scm.smartcontactmanager.entity.contact.Contact;
+import com.scm.smartcontactmanager.service.category.CategoryService;
+
+import jakarta.validation.Valid;
+
+@RestController
 public class CategoryController {
 
+	@Autowired
+	private CategoryService categoryService;
+	
+
+	@PostMapping(value = "/v1/category", consumes = "application/json", produces = "application/json")
+	public ResponseEntity<?> createCategory(@Valid @RequestBody CategoryRequestDto categoryRequestDto,
+			BindingResult bindingResult) {
+		if (bindingResult.hasErrors()) {
```

Comment:
This is not the right way to handle bad request. Please handle it using global exception handler